### PR TITLE
Eliminate "v" prefix from samples version

### PR DESCRIPTION
**Pull request by folmos-at-orange**
See https://github.com/KhiopsML/khiops/issues/234

### CODE REVIEW

File: khiops/tools.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def kh_download_datasets_entry_point():
         description=f"Downloads the Khiops samples dataset to {samples_dir}.",
     )
     arg_parser.add_argument(
-        "-v", "--version", default="10.1.1", help="Sample datasets version"
+        "-v", "--version", default="10.2.0", help="Sample datasets version"
```
Comment (popescu-v) [inline khiops/tools.py:99]: I am not sure about keeping this default here, as we would always need to bump it to the latest release.

Comment (folmos-at-orange) [reply khiops/tools.py:99]: Maybe a global variable ? We need both default values because the tool may be called from a shell or from Python.

Comment (popescu-v) [reply khiops/tools.py:99]: Yes, I see your point. However, it seems problematic to me to hard-code this version, thusly needing to update this code each time we bump the release version of the samples.

What if we do not add a default version and just make this argument required, thus leaving on the consumers of the tool the responsibility of setting this correctly?

**Review by popescu-v — COMMENTED** [review]
See comment.

File: khiops/tools.py (hunk 1 above)
Comment (folmos-at-orange) [reply khiops/tools.py:99]: This is intended to be easy to use, so no to add an argument.

Comment (popescu-v) [reply khiops/tools.py]: OK, then this global variable would be used for setting both:
- `version` default value in `download_datasets`; and
- `-v` argument in the `kh_download_datasets_entry_point`.
Hence, I would close this thread as soon as this change is implemented.

**Review by popescu-v — APPROVED** [review]
LGTM

**Pull request merged**